bundle/dao/src/main/java/hu/uni/miskolc/iit/swtest/team3/dao/BookDaoJdbc.java

```diff
@@ -36,8 +37,14 @@ public BookDaoJdbc(RowMapper<Book> rowMapper) {
     }
 
     @Override
-    public int create(Book book) {
-        return namedParameterJdbcTemplate.update(INSERT, getSqlParameterSource(book));
+    public boolean create(Book book) {
```

Comment:
Guys, it looks like, its not a good idea.
This solution is messy (if something goes wrong better to throw exception, not returning 'false')
Is this really required?? It looks like you can test everything without it until now.

Or if you need to change it, change it for every method of all DAOs that return int